tests/test_core_functionality.py

```
@@ -0,0 +1,438 @@
+        """Test server constants are properly defined."""
+        assert CONNECTED == b'HTTP/1.1 200 Connection established\r\n\r\n'
+
+    def test_api_constants(self):
+        """Test API constants are properly defined."""
+        assert GRAB_PAUSE == 180
+        assert MAX_CONCURRENT_PROVIDERS == 3
+
+    def test_error_types_exist(self):
+        """Test that error types are properly imported."""
+        from proxybroker.errors import (
+            ProxyConnError, ProxyTimeoutError, BadResponseError,
```

Ruff (0.11.9)

Redefinition of unused `ProxyConnError` from line 15

Remove definition: `ProxyConnError`

(F811)